geopandas/geodataframe.py

```diff
@@ -836,7 +836,7 @@ def __geo_interface__(self):
         represents the ``GeoDataFrame`` as a GeoJSON-like
         ``FeatureCollection``.
 
-        This differs from `_to_geo()` only in that it is a property with
+        This differs from `to_geo_dict()` only in that it is a property with
```

Comment:
```suggestion
        This differs from :meth:`to_geo_dict` only in that it is a property with
```
to get a direct link in docs